monkeytypes/credentials/password.py

```diff
@@ -8,3 +8,7 @@ class Password(InfectionMonkeyBaseModel):
 
     def __hash__(self) -> int:
         return hash(self.password)
+
+    @field_serializer("password", when_used="json")
+    def dump_secret(self, v):
+        return v.get_secret_value()
```

Comment:
Double check for all of these credentials that we have a unit test that verifies proper serialization/deserialization.